app/api/entities/entities.js

```diff
@@ -347,6 +347,16 @@ const withDocuments = (entities, documentsFullText, withPdfInfo) =>
     })
   );
 
+const reindexEntitiesByTemplate = async (template, options) => {
+  const templateHasRelationShipProperty = !template.properties?.find(
```

Comment:
Should this be named 'templateHasNorelationshipProperty' ?

Reply:
Thanks, I didn't realize that, instead  I extracted the negation, I think it's more readable